be/src/olap/tablet.cpp

```diff
@@ -206,6 +206,24 @@ OLAPStatus Tablet::revise_tablet_meta(const TabletMeta& tablet_meta,
         _tablet_meta = new_tablet_meta;
     } while (0);
 
+    for (auto& version : versions_to_delete) {
+        auto it = _rs_version_map.find(version);
+        _rs_version_map.erase(it);
```

Comment:
_inc_rs_version_map need also erase it?

Reply:
I have missed to handle _inc_rs_version_map.